src/components/React/Controls.js

```diff
@@ -1,7 +1,5 @@
 import React, { Component } from "react";
-
 import Slider from "rc-slider";
```

Comment:
is the `rc-slider` really needed? is there a react/bootstrap version that could be used?

Reply:
looks like the rc-slider does the two values which bootstrap doesn't include out of the box